.github/workflows/solvers-publish.yml

```diff
@@ -0,0 +1,47 @@
+# based on https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-a-registry-using-a-personal-access-token
+name: Push solvers package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - feat-packages
```

Comment:
this is needed until the workflow is merged to `main`, can't manually dispatch from a branch